index.html

```diff
@@ -21,6 +21,33 @@
 		<meta name="theme-color" content="#000000">
```

Comment:
what theme do you have because I `want` what you have.

Reply:
uh... some sort of blu i can put it in when we're done with all the code

Reply:
NO I WANT IT NOW!

Reply:
`'^'`